CHANGELOG.md

```diff
@@ -8,5 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 * BAM file merging doesn't happen if only a single BAM file is produced
 * Replace bamqc with smallBamQc
 
+## 1.1.0 - 2024-06-25
+[GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - add vidarr labels to outputs (changes to medata only)
```

Comment:
Put this under the `Unreleased` section. emSeqQc isn't ready for release yet, as the smallBamQc change requires more upstream fixes.

Reply:
sure